client/src/app/UnmuteMessage/UnmuteMessage.js

```diff
@@ -1,22 +1,38 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { motion } from 'framer-motion'
+import { ArrowIcon } from '../Icons/Icons'
 
 UnmuteMessage.propTypes = {
   children: PropTypes.string,
 }
 
+const variants = {
+  open: { opacity: 1, y: 0 },
+  closed: { opacity: 0, y: '-100%' },
+}
+
 export default function UnmuteMessage() {
+  const [isShown, setIsShown] = useState(true)
+
   return (
-    <MessageWrapper>
-      <p>Please unmute your device.</p>
+    <MessageWrapper animate={isShown ? 'open' : 'closed'} variants={variants}>
+      <p onClick={() => setIsShown(false)}>Please unmute your device.</p>
+      <Arrow />
     </MessageWrapper>
   )
 }
 
-const MessageWrapper = styled.section`
+const MessageWrapper = styled(motion.section)`
   background: var(--white-transparent-min);
+  border-radius: 20px;
+  box-shadow: var(--shadow-light);
+  display: flex;
+  justify-content: center;
   padding: 8px;
-  width: 100%;
+  margin: 20px auto;
+  width: 80%;
   p {
```

Comment:
You could name ``p`` e.g. ``Message``, to make it  a bit more "readable", if you want.